acf-to-wp-api.php

```diff
@@ -76,13 +76,26 @@ function getACFOptions() {
 	}
 
 	/**
+	 * Option/<name> endpoint
+	 * @param $name
+	 * @return mixed
+	 */
+	function getACFOption($name) {
+		return get_field($name, 'option');
+	}
+
+	/**
 	 * Custom routes
 	 */
 	function registerRoutes( $routes ) {
-		$routes['/option'] = array(
+		// all the options
+		$routes['/options'] = array(
```

Comment:
Changing from ```/option``` to ```/options``` breaks backwards compatibility. 
Maybe either add both or keep the old one?